src/PrestaShopBundle/Form/Admin/Extension/AlertExtension.php

```diff
@@ -80,8 +80,8 @@ public function buildView(FormView $view, FormInterface $form, array $options)
     /**
      * {@inheritdoc}
      */
-    public function getExtendedType()
+    public static function getExtendedTypes(): iterable
```

Comment:
It's a BC break from Symfony, not sure we have to document them all

Reply:
Not all but just indicate in the PR description that the extensions must now respect this new interface

Reply:
@jolelievre service definition doesn't require to repeat this info anymore ;) https://symfony.com/blog/new-in-symfony-4-2-improved-form-type-extensions
I think we can get rid of them in another PR.

I don't think this is a BC break because it's only deprecated. But I can add it to the noticeable changes though!

Reply:
Yep ok for noticeable changes!! Because I'm surprised it didn't cause any bugs, when I go in my Editor I get errors stating that the extension doesn't respect the interface (before your changes)

So it means if modules use form extension they need to adapt to this new way of implementing them.